srcpkgs/sharenix/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'sharenix'
+pkgname=sharenix
+version=0.10.0a
+revision=1
+build_style=go
+go_import_path="github.com/Francesco149/sharenix"
+hostmakedepends="pkg-config git"
+makedepends="gtk+-devel"
+depends="gtk+ gnome-screenshot"
+short_desc="ShareX clone for linux and FreeBSD"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Francesco149/sharenix"
+distfiles="https://github.com/Francesco149/sharenix/archive/${version}.tar.gz"
+checksum=45f00008b3e717d8d11ffd97d91811539e335f618f878c65bc8e2c9080618178
+
+post_install() {
+	vlicense LICENSE
```

Comment:
Don't need to install yet another copy of the GPL license, the standard ones are already in `/usr/share/licenses`.